src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -232,8 +232,8 @@ void OnOffServer::initOnOffServer(chip::EndpointId endpoint)
         // 0xff       Set the OnOff attribute to its previous value.
 
         // Initialize startUpOnOff to No action value 0xFE
```

Comment:
This comment no longer makes sense to have.

Reply:
Indeed, will adjust the comment.